src/scripts/anti-crise-scrapper.py

```diff
@@ -0,0 +1,112 @@
+import datetime
+from bs4 import BeautifulSoup
+from datetime import timedelta
+from src.models.Promotion import Promotion
+from src.scripts.utils import simple_get, promotion_post
+from src.scripts.scrapper import Scrapper
+
+USER_ID = 1
+HISTORY_FILE = "../misc/history"
```

Comment:
Relative paths are not great. You should can import Path from `pathlib` and do something like `Path(__file__).parent / 'misc/history'`